Formula/ruby@1.9.rb

```diff
@@ -44,6 +44,10 @@ def api_version
     "1.9.1"
   end
 
+  def rubygems_bindir
+    Pathname.new("#{HOMEBREW_PREFIX}/lib/ruby/gems/#{api_version}/bin")
```

Comment:
irb(main):001:0> HOMEBREW_PREFIX.class
=> Pathname

Reply:
Indeed. I think it's because we've tucked it inside the `"` rather than doing `HOMEBREW_PREFIX/"blah"`.

Reply:
why not do HOMEBREW_PREFIX/"blah"

Reply:
Heh, beat you to it 😉. My initial dance was just me slapping on the most blunt fix. I'll tweak it to the more reasonable syntax before this goes anywhere.

Reply:
I think I'll stick "do not merge" on this so no one gets overeager.

Reply:
Aye 👍. It shouldn't take too long to iron out, and it's going to be pretty ugly whatever happens, but it can absolutely be less ugly than it currently is.

Reply:
Also all the versioned tests are blowing up, which I'll need to look at, given that wasn't an issue with the main `ruby` in the other PR 🙃.

Reply:
I think when that happens, we just make the test block a version test, right?

Reply:
Looking at the failures, which are consistent throughout, I'm guessing either I'm hitting something sandbox-related or I've done something stupid. Possibly both.

Reply:
Strangely, the test did not fail on @2.3

Reply:
Maybe a `bundler` compatibility thing then, in terms of how it has to be initialised. Interesting.

Reply:
probably because it's setting GEM_HOME …

Reply:
Lol wow. Missed that difference completely 😅.